WriteFreely-MultiPlatform.xcodeproj/project.pbxproj

```diff
@@ -849,6 +849,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		17DF32C424C87E6700BCE2E3 /* ShellScript */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
From my commit message:

> This prevents a build warning in Xcode 14 because SwiftLint doesn’t yet support input/output files for these script execution build phases. It’s fast enough, so running it on all files isn’t an issue.